admin.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask, render_template, jsonify, request
+from flask_cors import CORS
+from flask_pymongo import PyMongo
+from pymongo import MongoClient
+
+import db
+import json
+import datetime
+from bson import json_util, ObjectId
+from lib import errors
+from lib.utils import clean_json, error, expect_json, success
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "ttmadmin"
+CORS(app)
+mdb = MongoClient(db.MONGO_URL).db
+
+@app.route("/auth", methods=["POST"])
+@expect_json(password=str)
+def admin_auth(body):
+    if(body['password'] == "ttmadmin"):
+        return success("Authorized")

```

Comment:
Optimally on a success this should return some kind of token that should then be used in the Authorization header for future requests